common/directoryBlock/directoryBlock.go

```diff
@@ -20,16 +20,28 @@ import (
 
 var _ = fmt.Print
 
+// DirectoryBlock holds all the information related to a directory block, which covers a ~10 minute period of time.
+// A directory block is the fundamental unit of the Factom blockchain and contains two main data objects:
+// a header and its entries. The header simply contains information about where this directory block fits
+// within the larger block chain. While the entries contain specific information entered into the block chain.
+// Each directory block always contains an admin block (organizational information for the block chain), and
+// EC block (entry credit transactions occuring during the ~10mins), a Factoid block (Factoid transactions occuring
+// during the ~10mins), and finally an unknown number of standard entry blocks (Each Entry Block contains all of the
+// entries for a particular Chain during a 10 minute period)
 type DirectoryBlock struct {
-	//Not Marshalized
-	DBHash     interfaces.IHash `json:"dbhash"`
-	KeyMR      interfaces.IHash `json:"keymr"`
-	HeaderHash interfaces.IHash `json:"headerhash"`
-	keyMRset   bool             `json:"keymrset"`
+	//Not Marshalized - hashing information related to the main entries below
+	DBHash     interfaces.IHash `json:"dbhash"`     // Hash of the 'body' of the DirctoryBlock (entries only)
+	KeyMR      interfaces.IHash `json:"keymr"`      // Merkle root of the HeaderHash and DBHash combined
+	HeaderHash interfaces.IHash `json:"headerhash"` // Hash of the header of the directory block
+	keyMRset   bool             `json:"keymrset"`   // has the key Merkle Root already computed
```

Comment:
The keyMRset is a run time bool, so it should not have a json tag. 
The other fields might be useful outside the directory block, but it is worth noting that if the "not Marshalized" fields are not meant to go out into the JSON, then they should have `json:"-"

We don't marshal the directory block as json, so I am not sure of the use case

Reply:
I have changed all the "not marshalled" tags to `json:"-"` as you suggest, please check this is ok